src/main/java/com/manager/data/housing/housingsupportmanager/model/Council.java

```diff
@@ -1,54 +1,109 @@
 package com.manager.data.housing.housingsupportmanager.model;
 
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
 import javax.validation.constraints.NotBlank;
 
+@Entity(name = "councils")
```

Comment:
Minor point: Convention is to not use plural as name of tables (same as class name)